Dockerfile

```diff
@@ -11,23 +10,25 @@ RUN apt-get install -y software-properties-common
 
 # PostgreSQL apt server for newer PostgreSQL and PostGIS versions
 RUN APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
-RUN add-apt-repository "deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main"
+RUN add-apt-repository "deb http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main"
 
 # deadsnakes ppa for modern python versions
 RUN add-apt-repository ppa:deadsnakes/ppa && apt-get update
 
 RUN apt-get install -y git unzip wget sudo curl build-essential gettext \
-    python python-dev python-pip python-virtualenv \
-    python3.6 python3.6-dev \
-    python3.7 python3.7-dev \
-    python3.8 python3.8-dev \
-    python3.9 python3.9-dev \
     postgresql-client-common libpq-dev \
     postgresql-12 postgresql-12-postgis-3 \
     libmemcached11 libmemcachedutil2 libmemcached-dev libz-dev memcached \
     libproj-dev libfreexl-dev libgdal-dev gdal-bin \
     ffmpeg
 
+# Various python versions
+RUN apt-get install -y python3 python3-pip \
```

Comment:
Heb je dit met een reden in een aparte `RUN` gezet? Een aparte `RUN` maakt namelijk ook weer een nieuwe image layer.

Reply:
Ja, ik was wat aan het proberen met verschillende python-versies, dan was het veel sneller als het in een nieuwe layer staat. Kan ik weer terugdraaien, ik heb geen goed beeld van de overhead van een extra layer.

Reply:
Dat zou ik wel doen idd, over het algemeen is minder layers beter.